charts/nexus3/templates/statefulset.yaml

```
@@ -62,13 +62,10 @@ spec:
         - name: import-ca-certs
           image: {{ include "nexus3.image" .Values.jdkImage }}
           imagePullPolicy: {{ .Values.jdkImage.pullPolicy }}
+          {{- with .Values.rootSecurityContext }}
```

```suggestion
          {{- with .Values.initRootSecurityContext }}
```

I think this is a more descriptive name.